### ISSUE TITLE
Possible memory leak

### ISSUE THREAD

**Issue by freaksauce** [IonDen/ion.rangeSlider#220]
I have noticed that when a Chrome tab is active that includes an instance of the ion.rangeSlider the memory for that tab instance in Windows 7 task manager is increasing by about 500kb per second. When the tab is inactive the memory usage freezes until the tab becomes active again.

This memory leak can also be observed on your demo pages, eg. http://ionden.com/a/plugins/ion.rangeSlider/demo.html

**Comment by jontra** [comment]
I can also see this behavior on my page. Also I can see constant CPU usage (around 5%), even if the page is idle.

I think its this line:
```
this.raf_id = requestAnimationFrame(this.updateScene.bind(this));
```

Which constantly calls the updateScene function.

**Comment by IonDen** [comment]
Hello, i will check this.

**Comment by freaksauce** [comment]
Hi @IonDen did you manage to replicate this issue?

**Comment by freaksauce** [comment]
I think @jontra is correct, if you console.log inside updateScene it is constantly running, surely the requestAnimationFrame should only when you are dragging?

**Comment by IonDen** [comment]
Hi all, i am thinking about this problem. I can't just turn it off, because it is important to trace some variables even in background. To manage resize issue and to init slider in display: none containers.

**Comment by ozkey** [comment]
fix for this bug here
https://github.com/IonDen/ion.rangeSlider/pull/232

**Comment by IonDen** [comment]
Should be fixed in 2.0.7

**Comment by ozkey** [comment]
Thank you
